### PR TITLE
py-rnacocktail: a new package.

### PR DESCRIPTION
**Pull request by weijianwen**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-rnacocktail/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    for accurate and efficient RNA-Seq analysis."""
+
+    homepage = "http://bioinform.github.io/rnacocktail/"
+    url      = "https://github.com/bioinform/rnacocktail/archive/v0.2.2.tar.gz"
+
+    version('0.2.2', sha256='34aa0d1d7bd9d80303fe7dac5acc0519f7c1ed986397692588343d82ce45c7a5')
+
+    depends_on('python@2.7:2.8', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-pybedtools', type='run')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-rnacocktail/package.py]:
```suggestion
    depends_on('py-pybedtools', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-rnacocktail/package.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    depends_on('py-pysam', type='run')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-rnacocktail/package.py]:
```suggestion
    depends_on('py-pysam', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-rnacocktail/package.py (hunk 1 above)
Comment (weijianwen) [reply var/spack/repos/builtin/packages/py-rnacocktail/package.py]: Fixed.

File: var/spack/repos/builtin/packages/py-rnacocktail/package.py (hunk 2 above)
Comment (weijianwen) [reply var/spack/repos/builtin/packages/py-rnacocktail/package.py]: Fixed.

**Pull request merged**